aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -229,16 +243,15 @@ proptest! {
     #![proptest_config(ProptestConfig {
         // Cases are expensive, few cases is enough.
         // We will test a few more comprehensive tests more times, and the rest even fewer.
-        // when trying to stress-test, increase (to 200 or more), and disable result cache.
-        cases: 10,
-        result_cache: prop::test_runner::basic_result_cache,
+        cases: if STRESSTEST_MODE { 1000 } else { 20 },
+        result_cache: if STRESSTEST_MODE { prop::test_runner::noop_result_cache } else {prop::test_runner::basic_result_cache },
         .. ProptestConfig::default()
     })]
 
     #[test]
     fn test_aggregator_lifetime(test_env in arb_test_env(14), element_type in arb_agg_type(), use_type in arb_use_type()) {
         println!("Testing test_aggregator_lifetime {:?}", test_env);
-        let mut h = setup(test_env.executor_mode, test_env.aggregator_execution_enabled, 14);
+        let mut h = setup(test_env.executor_mode, if use_type == UseType::UseResourceGroupType { AggregatorMode::EnabledOnly } else { test_env.aggregator_execution_mode}, 14);
```

Comment:
why enabled only for using resource groups?

Reply:
hm, some stale reason, works now